glue/sales_val_flagging.py

```diff
@@ -603,7 +609,7 @@ def get_parameter_df(
         "run_filter": [run_filter],
         "iso_forest_cols": [iso_forest_cols],
         "stat_groups": [stat_groups],
-        "tri_stat_groups": [tri_stat_groups],
+        "sales_to_write_filter": [sales_to_write_filter],
```

Comment:
Adjusting parameter tables to remove the `tri_stat_groups` and add the `sales_to_write_filter` column. A plus of this is that now we have removed tri from being hard-coded as a column, making the table structure more amenable to future changes.